psi4/driver/driver_cbs.py

```diff
@@ -1623,22 +2186,40 @@ def cbs(func, label, **kwargs):
             optionstash.restore()
 
         # Fill in energies for subsumed methods
+        # For DFT, we need to "correct" the -fctl energy by subtracting the -nl component.
+        # The "DFT VV10 ENERGY" is by default = 0.0, even for DFAs without -nl
         if ptype == 'energy':
-            for wfn in VARH[mc['f_wfn']]:
+            for component in VARH[mc['f_wfn']]:
                 for job in JOBS_EXT:
-                    if (wfn == job['f_wfn']) and (mc['f_basis'] == job['f_basis']) and \
-                       (mc['f_options'] == job['f_options']):
-                        job['f_energy'] = core.variable(VARH[wfn][wfn])
+                    if (mc['f_wfn'] == job['f_wfn']) and (mc['f_basis'] == job['f_basis']) and \
+                       (component == job['f_component']) and (mc['f_options'] == job['f_options']):
+                        job['f_energy'] = core.variable(VARH[mc['f_wfn']][component])
+                        if component.endswith("fctl"):
+                            job['f_energy'] -= core.variable("DFT VV10 ENERGY")
```

Comment:
all `core.variable` turn into something like wfn.variable from a single run in DDD (https://github.com/psi4/psi4/pull/1351/files#diff-9e10a6f2a977cb3eb6f9c34dea0a6d54110f18bbe841bf9f116107485612e26fR1616), so variable calls are fine, so long as they're local to a single job, not taking some advantage of cross-job storage.